src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Statistics/BoundaryLevelControllerTests.cs

```diff
@@ -0,0 +1,178 @@
+#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Admin.Requests;
+using GovUk.Education.ExploreEducationStatistics.Admin.Tests.Fixture;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Statistics;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Data.Model.Database;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Reflection;
+using System.Threading.Tasks;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Models.GlobalRoles;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Tests.Security.Utils.ClaimsPrincipalUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Controllers.Api.Statistics;
+public class BoundaryLevelControllerTests(TestApplicationFactory testApp) : IntegrationTestFixture(testApp)
+{
+    [Fact]
+    public async Task ListBoundaryLevels_Success()
+    {
+        await TestApp.AddTestData<StatisticsDbContext>(context =>
+        {
+            context.BoundaryLevel.AddRange(
+                new()
+                {
+                    Id = 1,
+                    Created = DateTime.Now,
+                    Label = "Boundary Level 1",
+                    Level = GeographicLevel.Country,
+                    Published = DateTime.Now,
+                },
+                new()
+                {
+                    Id = 2,
+                    Created = DateTime.Now,
+                    Label = "Boundary Level 2",
+                    Level = GeographicLevel.Country,
+                    Published = DateTime.Now,
+                });
+        });
+
+        var client = TestApp
+            .SetUser(AuthenticatedUser(RoleClaim(RoleNames.BauUser)))
+            .CreateClient();
+
+        var response = await client.GetAsync("api/boundary-level");
+
+        var result = response.AssertOk<List<BoundaryLevelViewModel>>();
+
+        Assert.Multiple(
```

Comment:
We should make this the standard way of asserting - it is inefficient with other unit tests where you only see one failure at a time.